lib/webauto/web4cucumber.rb

```diff
@@ -74,15 +74,15 @@ def started?
     def browser
       return @browser if @browser && @browser.exists?
       firefox_profile = Selenium::WebDriver::Firefox::Profile.new
-      chrome_caps = Selenium::WebDriver::Remote::Capabilities.chrome()
+      chrome_opts = Selenium::WebDriver::Options.chrome
```

Comment:
Can we also remove `Selenium::WebDriver::Remote::Capabilities.safari()` to resolve following warning? 

```
02-10 16:54:49.791  2023-02-10 08:54:49 WARN Selenium [DEPRECATION] [:caps_browsers] Remote::Capabilities.safari is deprecated. Use Options.safari instead.
```

Reply:
We can do that in a follow up PR after we verified the changes work in both Jenkins CI and Prow CI.